app/models/guest.rb

```diff
@@ -1,4 +1,5 @@
 class Guest < ApplicationRecord
   belongs_to :user
   belongs_to :party
+  has_one :movie, through: :party
```

Comment:
Do we need to have a guest have a movie?

Reply:
Just in the view I'm iterating through a users guests(which is a little confusing because the guest in the join between that user and a party) so it's how we know that user is invited to that party vs when it's the party host.  So in view instead of current_user.guests.each {|guest| guest.party.movie.title} can just do guest.movie.title